presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/presto/etc/catalog/kafka/write_all_datatypes_json.json

```diff
@@ -0,0 +1,170 @@
+        "dataFormat": "iso8601"
+      },
+      {
+        "name": "c_timestamp_rfc2822",
+        "type": "TIMESTAMP",
+        "mapping": "j_timestamp_rfc2822",
+        "dataFormat": "rfc2822"
+      },
+      {
+        "name": "c_timestamp_custom",
+        "type": "TIMESTAMP",
+        "mapping": "j_timestamp_custom",
+        "dataFormat": "custom-date-time",
+        "formatHint": "MM/yyyy/dd H:m:s"
+      },
+      {
+        "name": "c_date_iso8601",
+        "type": "DATE",
+        "mapping": "j_date_iso8601",
+        "dataFormat": "iso8601"
+      },
+      {
+        "name": "c_date_custom",
+        "type": "DATE",
+        "mapping": "j_date_custom",
+        "dataFormat": "custom-date-time",
+        "formatHint": "yyyy/dd/MM"
+      },
+      {
+        "name": "c_time_milliseconds_since_epoch",
+        "type": "TIME",
+        "mapping": "j_time_milliseconds_since_epoch",
+        "dataFormat": "milliseconds-since-epoch"
+      },
+      {
+        "name": "c_time_seconds_since_epoch",
+        "type": "TIME",
+        "mapping": "j_time_seconds_since_epoch",
+        "dataFormat": "seconds-since-epoch"
+      },
+      {
+        "name": "c_time_iso8601",
+        "type": "TIME",
+        "mapping": "j_time_iso8601",
+        "dataFormat": "iso8601"
+      },
+      {
+        "name": "c_time_custom",
+        "type": "TIME",
+        "mapping": "j_time_custom",
+        "dataFormat": "custom-date-time",
+        "formatHint": "mm:HH:ss"
+      },
+      {
+        "name": "c_timestamptz_milliseconds_since_epoch",
+        "type": "TIMESTAMP WITH TIME ZONE",
+        "mapping": "j_timestamptz_milliseconds_since_epoch",
+        "dataFormat": "milliseconds-since-epoch"
+      },
+      {
+        "name": "c_timestamptz_seconds_since_epoch",
+        "type": "TIMESTAMP WITH TIME ZONE",
+        "mapping": "j_timestamptz_seconds_since_epoch",
+        "dataFormat": "seconds-since-epoch"
+      },
+      {
+        "name": "c_timestamptz_iso8601",
+        "type": "TIMESTAMP WITH TIME ZONE",
+        "mapping": "j_timestamptz_iso8601",
+        "dataFormat": "iso8601"
+      },
+      {
+        "name": "c_timestamptz_rfc2822",
+        "type": "TIMESTAMP WITH TIME ZONE",
+        "mapping": "j_timestamptz_rfc2822",
+        "dataFormat": "rfc2822"
+      },
+      {
+        "name": "c_timestamptz_custom",
+        "type": "TIMESTAMP WITH TIME ZONE",
+        "mapping": "j_timestamptz_custom",
+        "dataFormat": "custom-date-time",
+        "formatHint": "MM/yyyy/dd H:m:s"
+      },
+      {
+        "name": "c_timetz_milliseconds_since_epoch",
+        "type": "TIME WITH TIME ZONE",
+        "mapping": "j_timetz_milliseconds_since_epoch",
+        "dataFormat": "milliseconds-since-epoch"
+      },
+      {
+        "name": "c_timetz_seconds_since_epoch",
+        "type": "TIME WITH TIME ZONE",
+        "mapping": "j_timetz_seconds_since_epoch",
+        "dataFormat": "seconds-since-epoch"
```

Comment:
seconds-since-epoch do not encode the zone component, right?

Reply:
Correct. The reader reads the value and returns a type with TZ set to UTC.